hardhat.config.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-undef */
+const dotenv = require('dotenv');
+
+const dotenvResult = dotenv.config();
+
+if (dotenvResult.error) {
+  throw dotenvResult.error;
+}
+
+require('@nomiclabs/hardhat-ethers');
+require('@nomiclabs/hardhat-solhint');
+require('@nomiclabs/hardhat-truffle5');
+require('hardhat-gas-reporter');
+require('solidity-coverage');
+
+const {
+  PROVIDER_HOST,
+  PROVIDER_PORT,
+  PROVIDER_URL,
+  NETWORK,
+  MNEMONIC,
+} = process.env;
+const GWEI = 1000000000;
+
+// Ref - https://chainid.network/chains.json
+const ENV_CHAIN_IDS = {
+  mainnet: 1,
+  goerli: 1337,
+};
+
+module.exports = {
+  defaultNetwork: 'hardhat',
+  solidity: {
+    version: '0.6.11',
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 1500,
+      },
+    },
+  },
+  paths: {
+    sources: './contracts',
```

Comment:
I think these are already defaults. Lets not redefine config params unless they are different than the defaults.